packages/replayio/src/utils/async/logPromise.ts

```diff
@@ -56,16 +61,16 @@ export async function logPromise(
   const interval = !isDebugging ? setInterval(print, dots.interval) : undefined;
 
   try {
-    await promise;
-
-    status = STATUS_RESOLVED;
+    deferred.resolve(await promise);
   } catch (error) {
-    status = STATUS_REJECTED;
+    deferred.reject(error as Error);
   }
 
   clearInterval(interval);
 
-  print();
-
-  logUpdate.done();
+  try {
+    print();
+  } finally {
+    logUpdate.done();
+  }
```

Comment:
I wrapped this just in case - the `messages.failed` could like to rethrow the error or something and in such a case we should still call this.

Reply:
I don't care, since it doesn't hurt to have this, but that scenario sounds bizarre to me. This helper is just for printing; control flow should be handled separately (with the original promise)

Reply:
I agree and I'm not sure what I was thinking here :P I will revert this change cause if anything the whole thing should be wrapped with this `finally` block but it would indent the whole thing and I don't think it's really worth it